db/active_replicator_common.go

```diff
@@ -133,6 +140,109 @@ func newActiveReplicatorCommon(ctx context.Context, config *ActiveReplicatorConf
 	return &apr, nil
 }
 
+// Start starts the replicator, setting the state to ReplicationStateStarting and starting the status reporter.
+func (apr *activeReplicatorCommon) Start(ctx context.Context) error {
```

Comment:
```suggestion
func (a *activeReplicatorCommon) Start(ctx context.Context) error {
```

(and the other methods below)